documentation/src/main/asciidoc/getting_started/embedded_cache.adoc

```diff
@@ -10,19 +10,15 @@ The only thing you need to set up {brandname} is add it's dependencies to your p
 ==== Maven users
 If you are using Maven (or another build system like Gradle or Ivy which can use Maven dependencies), then this is easy. Just add:
 
-[source,xml]
 .pom.xml
-----
-<dependency>
-	<groupId>org.infinispan</groupId>
-	<artifactId>infinispan-embedded</artifactId>
-	<version>${infinispan.version}</version>
-</dependency>
-
-----
+include::topics/dependencies/ispn_embedded.adoc[]
 
 to the `<dependencies>` section of the POM.
+
+ifndef::productized[]
 You'll need to substitute `${infinispan.version}` for the version of {brandname} you wish to use.
```

Comment:
Won't they still need this? Or I guess we need to wait to see what the string placeholder is in the customized ispn_embedded.adoc?

Reply:
yeah, I need to rework this. I've been messing about with it and think we can substitute `${infinispan.version}` with a variable that substitutes with the correct value for each release.